### ISSUE TITLE
Add support for Tailwindcss in Stylelint configuration

### ISSUE THREAD

**Issue by ahangarha** [microverseinc/linters-config#179]
I see people face some issues with Stylelint while using Tailwindcss. Please either update the linter config to automatically ignore the errors related to Tailwindcss or let the programmer make such modifications or provide a better way to do this than adding some tweak to the code base to make the linter avoid the error.

I don't think it is a good idea to manipulate the codebase to prevent errors caused by the QA mechanism.

**Comment by topeogunleye** [comment]
Yes, I completely agree with @ahangarha points above, We(students) have been having some issues with using Tailwind CSS, and it would be great if support for tailwind CSS can be added to the config files

**Comment by awais-amjed** [comment]
Yes, I am also interested in using Tailwindcss, So it would be great if Microverse can provide a way around.

**Comment by bellom** [comment]
> I see people face some issues with Stylelint while using Tailwindcss. Please either update the linter config to automatically ignore the errors related to Tailwindcss or let the programmer make such modifications or provide a better way to do this than adding some tweak to the code base to make the linter avoid the error.
> 
> I don't think it is a good idea to manipulate the codebase to prevent errors caused by the QA mechanism.

@ahangarha - Thank you.
